core/src/main/scala/kafka/server/KafkaRequestHandler.scala

```diff
@@ -146,64 +147,84 @@ class BrokerTopicMetrics(name: Option[String]) extends KafkaMetricsGroup {
     case Some(topic) => Map("topic" -> topic)
   }
 
+  case class MeterWrapper(metricType: String, eventType: String) {
```

Comment:
Can this be generalized to allow other Metric types like `Histogram` ?

Reply:
Do you think we should? Considering it's just used for Meters right now, I think we don't need to prematurely generalize it.

Reply:
I think we can leave this for when we need it.